cdap-ui/app/cdap/components/PipelineList/DeployedPipelineView/index.js

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, { Component } from 'react';
+import NamespaceStore from 'services/NamespaceStore';
+import {MyPipelineApi} from 'api/pipeline';
+import {Observable} from 'rxjs/Observable';
+import T from 'i18n-react';
+import orderBy from 'lodash/orderBy';
+import IconSVG from 'components/IconSVG';
+import StatusMapper from 'services/StatusMapper';
+import PipelineTable from 'components/PipelineList/DeployedPipelineView/PipelineTable';
+
+require('./DeployedPipelineView.scss');
+
+const INFO_MAP = {
+  'cdap-data-pipeline': {
+    programType: 'workflows',
+    programName: 'DataPipelineWorkflow'
+  },
+  'cdap-data-streams': {
```

Comment:
nit: can use GLOBALS (e.g. `[GLOBALS.etlDataStreams]`). Maybe we can add `DataPipelineWorkflow` and `DataStreamsSparkStreaming` to GLOBALS as well?